renishawWiRE/wdfReader.py

```diff
@@ -328,8 +328,18 @@ def _parse_orgin_list(self):
             self.origin_list_header[i][3] = s
             # Last: the actual data
             # array = numpy.empty(self.count)
-            array = numpy.array([self.__read_type("double")
+            
+            # Time appears to be recorded as int64 in 100 nanosecond intervals
+            # Reference does not appear to be  Unix Epoch time  
+            # Set time[0] = 0 until timestamp reference can be determined
+            if self.origin_list_header[i][1] == DataType.Time:             
```

Comment:
@rodsmithMB  Thank you very much for providing the patch, since I didn't have any file with series scan.

For rigorousness I suggest check against `self.origin_list_header[i][2]` to see what renishaw displays as the unit for the timestamp. If it is some new values larger than 24 then we need to add a dummy for that in the `types.py`

Reply:
It is 24. Here's the output for a series of 10 measurements with 1 second accumulations:

[True,
 <DataType.Time: 11>,
 <UnitType.FileTime: 24>,
 'Time',
 array([0.        , 1.15914154, 2.19147682, 3.35397148, 4.45767021,
        5.49030304, 6.6538887 , 7.6967659 , 8.78872299, 9.84212303])]

Reply:
got it! thanks for looking into this issue. I believe the strange 100 ns time tick is from the windows flavor of timestamp https://docs.microsoft.com/en-us/dotnet/api/system.datetime.ticks?redirectedfrom=MSDN&view=net-5.0#System_DateTime_Ticks which differs from unix time stamp.

Although we should be able to use absolute time stamps in this case, I agree with your approach. I'll merge the PR.